jrpc/options_test.go

```diff
@@ -22,4 +35,13 @@ func TestOptions(t *testing.T) {
 
 	assert.Len(t, def.CustomHeadersKey, 2)
 	assert.Equal(t, def.ReadHeaderTimeout.String(), dur.String())
+	require.NotNil(t, def.CorsOptions)
+	assert.Len(t, def.CorsOptions.AllowedOrigins, 1)
+	assert.Equal(t, "*", def.CorsOptions.AllowedOrigins[0])
+	assert.Len(t, def.CorsOptions.AllowedMethods, 5)
+	assert.Equal(t, corsMethods, def.CorsOptions.AllowedMethods)
```

Comment:
Same here. Comparing two arrays also compares their lengths.
Please always keep the tests short and simple.